zim/plugins/__init__.py

```diff
@@ -736,6 +736,12 @@ def notebook_properties(self, notebook):
 
 		return properties
 
+	def supports_image_format(self, fmt):
+		'''
+		Check if an image format is supported by GDK.
+		'''
+		return fmt in (f.name for f in GdkPixbuf.Pixbuf.get_formats())
+
```

Comment:
Seems better to me to put this helper method in the more specialized `zim/plugins/base/imagegenerator.py` base class. Or otherwise as a function in `zim/gui/widgets.py` (like the  `rotate_pixbuf()`  funtion)